pkg/client/connector/install.go

```diff
@@ -532,7 +532,7 @@ func (ki *installer) ensureAgent(c context.Context, namespace, name, portName, a
 }
 
 func (ki *installer) waitForApply(c context.Context, namespace, name string, dep *kates.Deployment) error {
-	c, cancel := context.WithTimeout(c, 1*time.Minute)
+	c, cancel := context.WithTimeout(c, client.GetConfig(c).Timeouts.Apply)
```

Comment:
Small suggestion, could we include the config timeout ussed in the timeout error message so if people see failures, they know which timeout to tweak?